chain/jsonrpc/Cargo.toml

```diff
@@ -41,3 +41,4 @@ dump_errors_schema = ["near-rpc-error-macro/dump_errors_schema"]
 adversarial = ["near-jsonrpc-primitives/adversarial"]
 nightly_protocol = ["near-primitives/nightly_protocol"]
 sandbox = []
+protocol_feature_routing_exchange_algorithm = []
```

Comment:
Super high level question, but why is it a protocol feature? Intuitively, the way peers communicate with each other feels like it shouldn't affect protocol per-se? We don't actually look at the runtime protocol version here at all. 

I guess here we just piggy-back onto the protocol featue infrastructure, just to avoig introducing ad-hoc feature for this? Migh make sense to explain this in a comment then.

Reply:
`RoutingTableActor` was previously only enabled under `adversarial` flag. It's going to be enabled by default now. However, some of it's methods are only available when `protocol_feature_routing_exchange_algorithm` is enabled. Otherwise `jsonrpc` wouldn't be able to call them.